content/table-schema/contents.lr

```diff
@@ -42,21 +44,31 @@ To illustrate here's a classic spreadsheet table:
      valA  |   valB  |  valC   |   valD    <--- Row
      ...
 
-In JSON a table would be:
+In JSON, a table would be:
 
     [
       { "A": value, "B": value, ... },
       { "A": value, "B": value, ... },
       ...
     ]
 
-# Descriptor
+### Physical and logical representation
+
+In order to talk about the representation and processing of tabular data from text-based sources, it is useful to introduce the concepts of the *physical* and the *logical* representation of data.
+
+The *physical representation* of data refers to the representation of data as text on disk, for example, in a CSV or JSON file. This representation may have some *type* information (JSON, where the primitive types that JSON supports can be used) or not (CSV, where all data is represented in string form).
 
-A Table Schema consists of a descriptor. The descriptor MUST be a JSON `object` (JSON is defined in [RFC 4627][]).
+The *logical representation* of data refers to the "ideal" representation of the data in terms of primitive types, data structures, and relations, all as defined by the specification. We could say that the specification is about the logical representation of data, as well as about ways in which to handle to conversion of a physical representation to a logical one.
+
+In this document, we'll explicitly refer to either the *physical* or *logical* representation in places where it prevents ambiguity for those engaging with the specification, especially implementors.
+
+For example, `constraints` should be tested on the logical representation of data, whereas a property like `missingValues` applies to the physical representation of the data.
+
+# Descriptor
 
-It MUST contain a property `fields`. `fields` `MUST` be an array where each entry in the array is a field descriptor (as defined below). The descriptor `MAY` have the additional properties set out below and `MAY` contain any number of other properties (not defined in this spec).
+A Table Schema is represented by a descriptor. The descriptor `MUST` be a JSON `object` (JSON is defined in [RFC 4627](http://www.ietf.org/rfc/rfc4627.txt)).
```

Comment:
@pwalsh Super minor: Can we please keep out of line markdown link references rather than inlining 😄